pkg/products/monitoring/prometheusRules.go

```diff
@@ -164,22 +164,22 @@ func (r *Reconciler) newAlertsReconciler() resources.AlertReconciler {
 						Labels: map[string]string{"severity": "warning"},
 					},
 					{
-						Alert: "PVCStorageWillFillIn4Days",
+						Alert: "KubePersistentVolumeFillingUp",
```

Comment:
Should this and the alert on line `177` be called the same? should we maybe change this one to `KubePersistentVolumeFillingUpPredictiveLongRun` or something along those lines? to have a clear way to distinguish their purpose from their name? I realise that this one is a `warning` level and the other one is `critical` but since they're queries are different then maybe their names should be as well? wdyt @grdryn ?

Reply:
Good question. I think it's fine that they have the same name. The alerts come from upstream [here](https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/master/alerts/storage_alerts.libsonnet). The message annotation will give more clarity than a different name ever could IMHO.